.github/workflows/ci_cd_main.yml

```
@@ -32,8 +31,10 @@ jobs:
     name: "Deploy development documentation"
     runs-on: ubuntu-latest
     needs: [doc-build]
+    permissions:
+      contents: write
     steps:
-      - uses: ansys/actions/doc-deploy-dev@main
+      - uses: ansys/actions/doc-deploy-dev@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/doc-deploy-dev@main
```
Needs to be reverted before merging